tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -370,13 +370,20 @@ After Docker is installed, launch a Docker container with the TensorFlow binary
 image as follows.
 
 ```bash
-$ docker run -it -p 8888:8888 gcr.io/tensorflow/tensorflow
+$ docker run -d -p 8888:8888 --name="tensor" gcr.io/tensorflow/tensorflow
```

Comment:
If we're going to leave it running in daemon mode, we need to include a sample command to stop it. And probably one to remove it?